cmd/olm/main.go

```diff
@@ -37,53 +37,53 @@ const (
 
 // config flags defined globally so that they appear on the test binary as well
 var (
-	kubeConfigPath = flag.String(
```

Comment:
where did kubeconfig go?

Reply:
it's being surfaced as an option by the controller manager from controller-runtime -- they conflict if this is left here -- so we get the value from that package now.